src/app/parcels/parcelsTable/filters.ts

```diff
@@ -108,16 +108,21 @@ const buildDeliveryCollectionFilter = async (): Promise<ParcelsFilter<string[]>>
         key: "deliveryCollection",
         filterLabel: "Method",
         itemLabelsAndKeys: optionsSet.map((option) => [option.value, option.key]),
-        initialCheckedKeys: optionsSet.map((option) => option.key),
+        initialCheckedKeys: [],
         method: deliveryCollectionSearch,
     });
 };
 
 const buildLastStatusFilter = async (): Promise<ParcelsFilter<string[]>> => {
+    const labelForNoStatus = "No Status";
+
     const lastStatusSearch: ParcelsFilterMethod<string[]> = (query, state) => {
-        if (state.includes("None")) {
+        if (state.length === 0) {
+            // Default is to show everything that's not deleted
```

Comment:
When mock data can be setup for the Parcels table, then tests for this will be written under [VFB-286](https://softwiretech.atlassian.net/browse/VFB-286)